src/window/trade_prices.c

```diff
@@ -14,38 +15,106 @@
 #include "input/input.h"
 #include "window/advisors.h"
 
+
 static void draw_background(void)
 {
     window_draw_underlying_window();
 
     graphics_in_dialog();
 
+    int has_caravanserai = building_monument_working(BUILDING_CARAVANSERAI);
+    int has_lighthouse = building_monument_working(BUILDING_LIGHTHOUSE);
+    int land_policy = building_monument_module_type(BUILDING_CARAVANSERAI);
+    int sea_policy = building_monument_module_type(BUILDING_LIGHTHOUSE);
+
+    int has_land_trade_policy = has_caravanserai && land_policy && land_policy != TRADE_POLICY_3;
+    int has_sea_trade_policy = has_lighthouse && sea_policy && sea_policy != TRADE_POLICY_3;
+    int same_policy = land_policy == sea_policy;
+
+    int window_height = 11;
+    int line_buy_position = 230;
+    int line_sell_position = 270;
+    int numberMargin = 25;
```

Comment:
Don't use camel case. Set the variable to `number_margin`

Reply:
professional deformation